modules/openapi-generator/src/main/resources/python/model.mustache

```diff
@@ -25,9 +25,17 @@ from {{packageName}}.exceptions import ApiAttributeError
 {{#imports}}
 {{#-first}}
 
-def lazy_import():
+def lazy_import(zzz_class_name_to_class=dict()):
+    {{{.}}}
```

Comment:
dicts are mutable. If you modify tgiz dict it stays modified. For that reason it's a bad practice to set argument defaults to empty dict. The second time you run the function the dict will not be empty. How about moving the empty dict assignment into the function?